### PR TITLE
add verification views

### PR DESCRIPTION
**Pull request by pxpm**
This PR should be merged when https://github.com/Laravel-Backpack/CRUD/pull/5301 gets merged.

image

### CODE REVIEW

**Comment by tabacitu** [comment]
This probably needs a little love too. Card to be narrower. Spacing before&after the card itself.

What I would do is copy-paste the EXACT view from registration. And just change the card content. That way, when people move from the Register screen to the Verify screen, it's seamless. Only the card content seems to change.

**Comment by pxpm** [comment]
image

File: resources/views/auth/verify-email.blade.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+@extends(backpack_view('layouts.plain'))
+
+@section('content')
+
+<div class="row justify-content-center align-items-center d-flex flex-row min-vh-100">
+    <div class="col-12 col-md-8 col-lg-6">
+        <h3 class="text-center mb-4">{{ trans('backpack::base.verify_email.email_verification') }}</h3>
+        <div class="card p-4 mb-0">
```
Comment (tabacitu) [inline resources/views/auth/verify-email.blade.php:8]:
```suggestion
        <div class="card mb-0">
```

File: resources/views/auth/verify-email.blade.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            <div class="card-body">
+            {{ trans('backpack::base.verify_email.email_verification_required') }}
+
+            @if (session('status') == 'verification-link-sent')
+                <div class="mb-4 mt-2">
```
Comment (tabacitu) [inline resources/views/auth/verify-email.blade.php:13]:
```suggestion
                <div class="mt-3">
```

File: resources/views/auth/verify-email.blade.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                    <div class="alert alert-success" role="alert">
```
Comment (tabacitu) [inline resources/views/auth/verify-email.blade.php:14]:
```suggestion
                    <div class="alert alert-success my-0" role="alert">
```

**Review by tabacitu — CHANGES_REQUESTED** [review]

**Pull request merged**